test/testinput/bufrtest_bufrdescription.yaml

```diff
@@ -0,0 +1,77 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+inputpath: "/Users/rmclaren/Work/sample-bufr-data/gdas/gdas.20200704/12"
+outputpath: "/Users/rmclaren/Temp/"
+
+
+bufr:
```

Comment:
This dictionary should be more generic than `bufr` e.g. `raw_data` or something like that.
In this dictionary, you could provide a key e.g. `name: bufr` which will identify the parser for the `raw_data`.

Reply:
Covered in previous conversation.